### DISCUSSION TITLE
Cases Disappearing

### DISCUSSION THREAD

**Discussion by informalsecurity** [Security-Onion-Solutions/securityonion#7266] [General] [answered]
I have a new instance up and running for about 4 days now; twice in the past 4 days, all the Cases I have created have disappeared, as well as any alerts that I have escalated to be a part of those cases.
<img width="1281" alt="Screen Shot 2022-02-20 at 9 57 29 AM" src="https://user-images.githubusercontent.com/32649378/154848804-6dfc53ee-734b-4054-9a1d-8cb0cca598eb.png">

The first time it happened, I probably only had about 6 cases open.  This time I had about 15.  When I go to alerts to search for alerts that have been escalated - I am not getting anything back.  Is there some kind of auto-pruning feature that is deleting cases?  Is there somewhere I can look (specific database/table) to see if the data still exists - or is it lost forever.

**Comment by dougburks** [comment]
What is the output of the following?
```
sudo so-elasticsearch-indices-list |grep case
```
```
sudo grep -ri log_size_limit /opt/so/saltstack/local/pillar/
```
```
df -h
```

**Comment by informalsecurity** [comment]
<img width="1046" alt="Screen Shot 2022-02-20 at 2 30 06 PM" src="https://user-images.githubusercontent.com/32649378/154860630-f74e4d1d-e78c-42d3-a07c-c96700829bce.png">

**Comment by dougburks** [comment]
What is the output of the following?
```
zgrep "so-case" /opt/so/log/curator/*
```

I suspect that Curator is not excluding the case indices properly. If that's the case, we'll start working on a proper fix. In the meantime, a temporary option might be to increase `log_size_limit` in `somanager_managersearch.sls` to be a higher percentage of /nsm (depending on what other data you have there). Note that 126GB for /nsm is barely above the minimum requirements so you might want to increase that if possible:
https://docs.securityonion.net/en/2.3/partitioning.html

**Comment by informalsecurity** [comment]
<img width="1223" alt="Screen Shot 2022-02-20 at 2 49 25 PM" src="https://user-images.githubusercontent.com/32649378/154861437-13e94aae-2d07-4085-892a-82442099aa34.png">

**Comment by informalsecurity** [comment]
So if the index is being deleted - does that mean all the cases are completely gone?  Also - would the original escalated alerts still be available somewhere?

**Comment by dougburks** [comment, marked as answer]
I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/7270 to fix the Curator configuration.

It looks like Curator has deleted the cases. It's possible that some of the escalated alerts might still be there if you go to Alerts, click Options, and then click Escalated. However, it does look like Curator is having to delete fairly recent data to stay at your current `log_size_limit` value. Therefore, you'll most likely want to increase the size of your `/nsm` partition if at all possible.

**Comment by informalsecurity** [comment]
Working on it now; I appreciate the help and all the great things you guys do!

**Comment by dougburks** [comment]
We've merged a fix that will be included in the upcoming 2.3.110 release:
https://github.com/Security-Onion-Solutions/securityonion/pull/7273/files

**Comment by dougburks** [comment]
We've released an official hotfix for this:
https://blog.securityonion.net/2022/03/security-onion-23100-20220301-hotfix.html
